app/models/annotation/AnnotationService.scala

```diff
@@ -278,23 +282,28 @@ object AnnotationService
 
     val annotationsGrouped: Map[String, List[Annotation]] = annotations.groupBy(_.dataSetName)
     val tracings = annotationsGrouped.map {
-      case (dataSetName, annotations) => (getTracings(dataSetName, annotations.map(_.tracingReference)), getNames(annotations), getScales(annotations))
+      case (dataSetName, annotations) => (getTracings(dataSetName, annotations.map(_.tracingReference)), getNames(annotations), getScales(annotations), Fox.successful(annotations))
     }
     tracings.toList
   }
 
-  def flattenListToListMap(listToListMap: List[(Fox[List[SkeletonTracing]], Fox[List[String]], Fox[List[Scale]])]):
-  Fox[List[(SkeletonTracing, String, Scale)]] = {
+  def flattenListToListMap(listToListMap: List[(Fox[List[SkeletonTracing]], Fox[List[String]], Fox[List[Scale]], Fox[List[Annotation]])]):
+  Fox[List[(SkeletonTracing, String, Scale, Annotation)]] = {
+
+    def zippedFourLists[A,B,C,D](l1: List[A], l2: List[B], l3: List[C], l4: List[D]): List[(A, B, C, D)] = {
```

Comment:
@fm3 we talked about putting this method in "util" but it seems to be very specific in my eyes. Should I put it in "util" anyway?

Reply:
Putting generic stuff in util has the benefit of less code duplication if anyone thinks of needing this again. That said, I hope no one ever needs this function (again) :D so feel free to leave it here if you prefer.